src/sdk/auth.js

```diff
@@ -214,6 +220,6 @@ export class OCAuthSandbox extends OCAuthCore
         const storageClass = getStorageClass(opts);
         const pkceTransactionManager = new TransactionManager( storageClass );
         const tokenManager = new TokenManager( storageClass, tokenEndpoint, publicKey );
-        super( loginEndpoint, redirectUri, pkceTransactionManager, tokenManager, referralCode, logoutEndpoint );
+        super( loginEndpoint, redirectUri, pkceTransactionManager, tokenManager, referralCode, logoutEndpoint, logoutReturnTo );
```

Comment:
let's not do this as an option because there are not fixed. 
typically for an SSO sdk this is passed in as an 'optional parameter' for the logout method.